test/inductor/test_torchinductor_opinfo.py

```diff
@@ -298,7 +298,6 @@ def process(device_type):
     "linalg.lstsq.grad_oriented": {f32, f64},
     "linalg.matrix_rank": {f32, f64},
     "linalg.matrix_rank.hermitian": {f32, f64},
-    "linalg.pinv.singular": {f32, f64},
```

Comment:
Looks like line 219 should be removed as well

Reply:
yup agreed